projects/optic/src/utils/organization.ts

```diff
@@ -3,7 +3,7 @@ import { OpticBackendClient } from '../client';
 
 export async function getOrganizationFromToken(
   client: OpticBackendClient,
-  message: string
+  message: string | false
```

Comment:
Updates so that we can pass in false to just select the first org in certain cases (i.e. generated or CI)